edx_proctoring/callbacks.py

```diff
@@ -43,11 +42,15 @@ def start_exam_callback(request, attempt_code):  # pylint: disable=unused-argume
         mark_exam_attempt_as_ready(attempt['proctored_exam']['id'], attempt['user']['id'])
 
     log.info("Exam %r has been marked as ready", attempt['proctored_exam']['id'])
-    template = loader.get_template('proctored_exam/proctoring_launch_callback.html')
-
-    return HttpResponse(
-        template.render({
-            'platform_name': settings.PLATFORM_NAME,
-            'link_urls': settings.PROCTORING_SETTINGS.get('LINK_URLS', {})
-        })
-    )
+
+    course_id = attempt['proctored_exam']['course_id']
+    content_id = attempt['proctored_exam']['content_id']
+
+    try:
+        exam_url = reverse('jump_to', args=[course_id, content_id])
+    except NoReverseMatch:
+        log.exception("Can't find course info url for course %s", course_id)
+        exam_url = ''
+    response = HttpResponseRedirect(exam_url)
+    response.set_signed_cookie('exam', attempt['attempt_code'])
```

Comment:
Note: I did not set a max time on the cookie as Dave suggested, since that way lies the same madness I was trying to solve as in https://openedx.atlassian.net/browse/EDUCATOR-4219